docs/docs-content/getting-started/aws/scale-secure-cluster.md

```diff
@@ -15,7 +15,27 @@ scans.
 
 This tutorial will teach you how to use the Palette UI to perform scale and maintenance tasks on your clusters. You will
 learn how to create Palette projects and teams, import a cluster profile, safely upgrade the Kubernetes version of a
-deployed cluster and scale up your cluster nodes.
+deployed cluster and scale up your cluster nodes. The concepts you learn about in the Getting Started section are
+centered around a fictional case study company, Spacetastic Ltd.
+
+## 🧑‍🚀 Back at Spacetastic HQ
+
+The team have been impressed with Palette's capabilities and decide to become a Spectro Cloud customer. The last piece
+of the puzzle is to learn how to handle Day-2 operations, which become increasingly more important as the Spacetastic
+platform matures. They must ensure that their systems are patched, upgraded, scaled, and scanned for vulnerabilities.
+These maintenance tasks must be automated and applied on a schedule, as the entire team wants to focus on providing
+Spacetastic features.
+
+> "I've read your report on Palette adoption at Spacetastic." says Meera, who provides the security expertise at
+> Spacetastic. I was impressed with the ability to roll out updates to all clusters using the same cluster profile. This
+> will streamline our system upgrades and cluster patching. Keeping up with security best practices has never been more
+> important, now that we are growing faster than ever!"
+>
+> "I agree. No matter how safe our coding practices are, we need to periodically review, patch and upgrade our
+> dependencies." says Wren, who leads the engineering team at Spacetastic.
+>
+> Kai nods, scrolling through the Spectro Cloud Docs. "Team, Palette has more security and Day-2 operation support than
```

Comment:
```suggestion
> Kai nods, scrolling through the Palette Docs. "Team, Palette has more security and Day-2 operation support than
```

I think Palette docs makes more sense considering that's what the docs are for 😅